frontend/src/metabase/lib/errors/is-static-embedding-entity-loading-error.ts

```diff
@@ -0,0 +1,13 @@
+import type { StaticEmbeddingEntityError } from "metabase/lib/errors/types";
+
+export const isStaticEmbeddingEntityLoadingError = (
+  error: unknown,
+): error is StaticEmbeddingEntityError => {
+  return (
+    !!error &&
+    typeof error === "object" &&
+    "status" in error &&
+    "data" in error &&
+    typeof error.data === "string"
```

Comment:
I don't see any condition here that clearly identifies it as an "StaticEmbeddingEntityError". I was expecting some field to disambiguate the error from other error types. I'm a bit worried if this is used along other error predicates that it'd fall into this `isStaticEmbeddingEntityLoadingError` check incorrectly...